File: server/api/v1/lcpblogapi/Models/Schedules.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace lcpblogapi.Models;

public class Schedules {
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    [Key]
    public int? ScheduleId { get; set; }

    [DefaultValue("Event1")]
    public string Title { get; set; } = null!;

    public DateTime DateStart { get; set; }
    public DateTime DateEnd { get; set; }

    [DefaultValue(false)]
    public bool? AllDay { get; set; }
}

File: src/app/[locale]/interfaces/schedules.ts
export interface Schedules {
    scheduleId: number;
    title: string;
    dateStart: Date | string;
    dateEnd: Date | string;
    allDay?: boolean;
}

export interface SchedulesEvents {
    scheduleId: number;
    title: string;
    start: Date | string;
    end: Date | string;
    allDay?: boolean;
}

export interface SchedulesData {
    events: Schedules[];
}